CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.2.2)
 project(leatherman VERSION 0.12.0)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+# Populate locale install location
+configure_file(cmake/leatherman.cmake.in "${CMAKE_CURRENT_SOURCE_DIR}/cmake/leatherman.cmake" @ONLY)
```

Comment:
So I see us using PROJECT_BINARY_DIR here: https://github.com/puppetlabs/leatherman/blob/master/CMakeLists.txt#L125-L126. According to the docs, CMAKE_BINARY_DIR is the top level of build tree (i.e. == CMAKE_SOURCE_DIR in our use case), while PROJECT_BINARY_DIR is the build directory. Are the two file generation use cases here different and therefore should use different targets, or did you mean one or the other?